db-scripts/src/main/resources/migration.sql

```diff
@@ -849,4 +849,23 @@ CREATE TABLE `resource_study` (
   PRIMARY KEY (`INTERNAL_ID`, `RESOURCE_ID`, `URL`),
   FOREIGN KEY (`INTERNAL_ID`) REFERENCES `cancer_study` (`CANCER_STUDY_ID`) ON DELETE CASCADE
 );
-UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.4";
\ No newline at end of file
+UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.4";
+
+##version: 2.12.5
+-- survival data migration
+-- create temporary table to store survival attributes
+CREATE TEMPORARY TABLE IF NOT EXISTS survival_attributes AS 
```

Comment:
Given that a temporary table is created, I don't think the "If not exists" clause is required, but it doesn't hurt.